src/Sentry/Internal/Hub.cs

```diff
@@ -20,6 +20,9 @@ internal class Hub : IHub, IDisposable
 
     internal IInternalScopeManager ScopeManager { get; }
 
+    /// <inheritdoc cref="IMetricAggregator"/>
+    public IMetricAggregator Metrics { get; }
```

Comment:
Yeah, it's on the interface:

https://github.com/getsentry/sentry-dotnet/blob/c65355dede946090646be759d5a39f3df19fd4fb/src/Sentry/ISentryClient.cs#L75

Reply:
I think it has to be on the interface, otherwise the SentrySdk can't get at it here:
https://github.com/getsentry/sentry-dotnet/blob/36fe0291aa87ca4e561953c47d3c75daf5ad19fe/src/Sentry/SentrySdk.cs#L621-L623